src/components/Cards.js

```diff
@@ -3,15 +3,11 @@ import React from 'react';
 export default function Cards(props) {
     return (
         <div className="card products-cards m-2 shadow">
-            <img src={props.img} className="card-img-top shadow-sm" alt={props.title}></img>
+            <img src={props.images.path} className="card-img-top shadow-sm" alt={props.images.title}></img>
```

Comment:
are you sure this is compatible with the current schema here https://github.com/aldoestebanpaz/rc-shop2-backend-com4/blob/master/products.js?